mnd/mnd/registry/patients/mnd_admin_forms.py

```diff
@@ -31,23 +42,41 @@ def _clean_fields(self):
         ndis_number_set = self.data.get(self.field_name('ndis_number'), '') != ''
         self.fields['ndis_plan_manager'].required = ndis_number_set
         ndis_coordinator_info = [
-            'ndis_coordinator_first_name', 'ndis_coordinator_last_name', 'ndis_coordinator_phone'
+            'ndis_coordinator_first_name', 'ndis_coordinator_last_name', 'ndis_coordinator_phone',
+            'ndis_coordinator_email'
         ]
         coordinator_required_data = self.data.get(self.field_name('ndis_plan_manager'), '') == 'other'
         for f in ndis_coordinator_info:
             self.fields[f].required = coordinator_required_data and ndis_number_set
+        dva_card_number_set = self.data.get(self.field_name('dva_card_number'), '') != ''
+        self.fields['dva_card_type'].required = dva_card_number_set
+        referred_for_mac_care_set = self.data.get(self.field_name('referred_for_mac_care'), '') != '3'
```

Comment:
`referred_for_mac_care` is a `BooleanField`. Why are we comparing it to `'3'`?

Reply:
Since it's a nullable boolean field django generates a select with 3 options: Unknown with value "1",  "Yes" with value "2" and "No" with value "3".

Reply:
OK. Then wouldn't this be set if it is `== 1`?
And could we declare a constant to make the code more obvious?
ex. `NULLABLE_BOOLFIELD_YES_CHOICE` or `NULLABLE_BOOLFIELD_CHOICES.YES` or similar

Reply:
Updated.